<launch>

<arg name="mav_name" default="alphasense" />
<arg name="namespace" default="$(arg mav_name)" />
<arg name="input_camera_name" default="cam4" />
<arg name="scale" default="1.0" />
<!-- <arg name="calib_path" default="$(find mav_startup)/parameters/mavs/elster/camchain_p23037.yaml"/> -->
<arg name="calib_path" default="/home/gabriel/catkin_ws/src/image_undistort/cfg/alphasense_cam4.yaml"/>

<group ns="$(arg namespace)">
  <node name="image_undistort_node" pkg="image_undistort" type="image_undistort_node" output="screen" clear_params="true">
    <param name="input_camera_namespace" value="$(arg input_camera_name)"/>      
    <param name="input_camera_info_from_ros_params" value = "true"/>    
    <param name="scale" value="$(arg scale)"/>
    <rosparam file="$(arg calib_path)"/>
    <remap from="input/image" to="/darknet_ros_camera_right/uncompressed"/> <!--"/alphasense_driver_ros/$(arg input_camera_name)"/-->
    <remap from="input/bounding_box" to="/darknet_ros_camera_right/bounding_boxes"/> <!--"/alphasense_driver_ros/$(arg input_camera_name)"/-->
  </node>
</group>

</launch>